packages/config/src/index.js

```diff
@@ -8,7 +10,7 @@ const { normalizeConfig } = require('./normalize')
 const { handleFiles } = require('./files')
 const { EVENTS, LEGACY_EVENTS } = require('./events')
 
-const resolveConfig = async function(configFile, { cwd, ...options } = {}) {
+const resolveConfig = async function(configFile, { cwd = getCwd(), ...options } = {}) {
```

Comment:
Nitpick: shouldn't need to change this file since `getConfigPath` already has this behavior.

Reply:
At the moment, `cwd` is only used inside `getConfigPath()`. However in the future, this option might be used elsewhere.

The reason it is also in `getConfigPath()` (and not only top-level) is because that method is exported and used in the CLI.

Reply:
Ah! Good reason 👍